build/build-constants.yaml

```diff
@@ -81,15 +81,6 @@
     - type: shell
       directory: build
       file-name-prefix: __
-- name: file-paths
```

Comment:
why remove the whole section? we can use it to hold the value `oryx-manifest.toml`, as it is used in both C# and Go code

Reply:
Good point. Will change it.